lib/Styliner.js

```diff
@@ -124,6 +123,17 @@ function parseDataUri(uri) {
 		return decodeURIComponent(parsed[3]);
 }
 
+/**
+ * Asynchronously parses an HTML document from file path passed as argument and inlines styles as appropriate.
+ *
+ * @param {String}			filePath				The file path to parse HTML document
+ */
+Styliner.prototype.parseFile = function(filePath) {
+	var _this = this;
+	return  qfs.read(qfs.resolve(qfs.directory(filePath), filePath)).then(function(content) {
+		return _this.processHTML(content);
```

Comment:
You still need to pass the directory containing the file as the second parameter.
Without that, relative paths in the HTML will not work.